### ISSUE TITLE
Forms / Labels / Associating labels implicitly

### ISSUE THREAD

**Issue by holloway** [w3c/wai-tutorials#605]
In [Labeling Controls](https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly) is the advice,

>  Generally, explicit labels are better supported by assistive technology.

Which assistive technology? Which versions?

I found [this earlier issues](https://github.com/w3c/wai-tutorials/issues/416), but it would be good to know when or if this advice is outdated.

**Comment by yatil** [comment]
Hi @holloway,

The advice is vague because the assistive technology market is changing quickly and is very diverse. Keeping exact track of support is not in scope for these tutorials (if even feasible). As far as I know Dragon still either does not support implicit labels or struggles with them. Using for/id also provides a clear association that is much more robust when it comes to ease of implementation in new assistive tech but also when the underlying HTML code changes or breaks.

I’ll see how I can integrate that info into the tutorials in the future without being too wordy or distracting.
